Makefile

```diff
@@ -145,6 +145,18 @@ FC_VER := $(FC_VER_MAJOR).$(FC_VER_MINOR).$(FC_VER_PATCH)
 LD_FLAGS        :=
 EXTRA_LD_FLAGS  :=
 
+#
+# Setup locale
+#
+LOCALE_LIST := en da
+ifeq ($(LOCALE),)
+LOCALE := en
+endif
+ifeq ($(filter $(LOCALE),$(LOCALE_LIST)),)
+    $(error LOCALE $(LOCALE) must be one of >$(LOCALE_LIST)<)
+endif
+INCLUDE_DIRS += $(INCLUDE_DIRS) $(SRC_DIR)/locales/$(LOCALE)
```

Comment:
Change to put the locales outside of main.
```
src/locales
```

Reply:
Well, I cosider from start, but you will have source placed like ../locales/ as the only one, also SRC_DIR are src/main etc, but if you want, no issue.

Reply:
All good - I was not fussed either way. The only advantage would be if we want to "autogenerate" these in the future, means we could potentially git ignore the directory (outside of main), and rebuild repeatedly using make, so we can update without updating the BF version.

Reply:
I think, in case of firmware development where size and speed matters, the developer need to be in charge of changes, and in the way of PRs. Fx the function cliPrintInternal() made the pgm to went down when met a char STR_XX[] variable, and this kind of issues are not a job for a translator to discover in firmware.
So to use Crowdin to translation, which are smart, I think a tool who take TR* in location.c as key for lookup in crowdin XML file and get locale and translated string to write/replace in xx/locale.h will be the best way.
In Crowdin the key then will be TR_something and it's translated string.
For now I keep in main/locales :-)